src/controllers/itemDetails/index.js

```diff
@@ -1095,7 +1095,7 @@ function renderMoreFromSeason(view, item, apiClient) {
             section.classList.remove('hide');
             section.querySelector('h2').innerText = globalize.translate('MoreFromValue', item.SeasonName);
             const itemsContainer = section.querySelector('.itemsContainer');
-            cardBuilder.buildCards(result.Items, {
+            cardBuilder.buildCards(result.Items.filter((i) => i.Id !== item.Id), {
```

Comment:
Let's move the filtering to be at the start of the function in the `.then()`.

```ts
const items = result.Items.filter(i => i.Id !== item.Id);
```

The if statement checking the length will need updated also.